accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/impl/AcInstallationServiceImpl.java

```diff
@@ -214,8 +214,7 @@ public void installConfigurationFiles(PersistableInstallationLogger installLog,
             LOG.info("Successfully applied AC Tool configuration in " + msHumanReadable(executionTime));
             installLog.setExecutionTime(executionTime);
         } catch (Exception e) {
-            // TODO: separate exception
-            installLog.addError(e.toString()); // ensure exception is added to installLog before it's persisted in log in finally clause
+            installLog.addError("Could not process yaml files", e); // ensure exception is added to installLog before it's persisted in log in finally clause
```

Comment:
It would be good to pass in `LOG` here as well?

Reply:
No need, as the exception is thrown in the next line anyways!